### DISCUSSION TITLE
Terminal Browser edit is broken w3m

### DISCUSSION THREAD

**Discussion by akashpal-21** [xwmx/nb#280] [Q&A] [answered]
If I remember correctly, I was able to edit in $EDITOR and drop back to w3m to save earlier, but now it just shows a borked version of the text which is not editable anymore, is this a bug/regression or am i missing something?

**Comment by akashpal-21** [comment, marked as answer]
Nevermind, I have to press "a" inside the box for the edit screen to open apparently...

**Reply by xwmx** [reply]
Glad you figured it out. On my system the `$EDITOR` can also be opened by double clicking the textarea in w3m.
